libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/YamlConfigService.java

```diff
@@ -64,8 +79,9 @@ private void addConfig(Map<String, String> map, JsonNode tree, String prefix) {
     @Override
     public String getParameter(String path) {
         if (getFeatureSet() != null) {
-            for (String featureSet : getFeatureSet()) {
```

Comment:
This was a SonarCloud request, as it would hide the higher-level featureSet attribute